components/custom_status/custom_status_modal.tsx

```diff
@@ -0,0 +1,284 @@
+            />
+        ) : (
+            <EmojiIcon className={'icon icon--emoji'}/>
+        );
+
+    const clearHandle = () => {
+        setText('');
+        setEmoji('');
+    };
+
+    const disableSetStatus = (currentCustomStatus.text === text && currentCustomStatus.emoji === emoji) ||
+        (text === '' && emoji === '');
+
+    const handleSuggestionClick = (status: any) => {
+        setEmoji(status.emoji);
+        setText(status.text);
+    };
+
+    const calculateRightOffSet = () => {
+        let rightOffset = Constants.DEFAULT_EMOJI_PICKER_RIGHT_OFFSET;
+        const target = getCustomStatusControlRef();
+        if (target) {
+            const anyTarget: any = target;
+            rightOffset = window.innerWidth - anyTarget.getBoundingClientRect().left - EMOJI_PICKER_WIDTH_OFFSET;
+            if (rightOffset < 0) {
+                rightOffset = Constants.DEFAULT_EMOJI_PICKER_RIGHT_OFFSET;
+            }
+        }
+
+        return rightOffset;
+    };
+
+    const recentStatuses = (
+        <div>
+            <div className='statusSuggestion__title'>
+                {formatMessage({id: 'custom_status.suggestions.recent_title', defaultMessage: 'RECENT'})}
+            </div>
+            {
+                recentCustomStatuses.map((status: UserCustomStatus) => (
+                    <CustomStatusSuggestion
+                        key={status.text}
+                        handleSuggestionClick={handleSuggestionClick}
+                        handleClear={handleRecentCustomStatusClear}
+                        emoji={status.emoji}
+                        text={status.text}
+                    />
+                ))
+            }
+        </div>
+    );
+
+    const renderCustomStatusSuggestions = () => {
+        const recentCustomStatusTexts = recentCustomStatuses.map((status: UserCustomStatus) => status.text);
+        const customStatusSuggestions = defaultCustomStatusSuggestions.
+            map((status) => ({
+                emoji: status.emoji,
+                text: formatMessage({id: status.message, defaultMessage: status.messageDefault}),
+            })).
+            filter((status: UserCustomStatus) => !recentCustomStatusTexts.includes(status.text)).
+            map((status: UserCustomStatus, index: number) => (
+                <CustomStatusSuggestion
+                    key={index}
+                    handleSuggestionClick={handleSuggestionClick}
+                    emoji={status.emoji}
+                    text={status.text}
+                />
+            ));
+
+        if (customStatusSuggestions.length > 0) {
+            return (
+                <>
+                    <div className='statusSuggestion__title'>
+                        {formatMessage({id: 'custom_status.suggestions.title', defaultMessage: 'SUGGESTIONS'})}
+                    </div>
+                    {customStatusSuggestions}
+                </>
+            );
+        }
+
+        return null;
```

Comment:
please switch the if-clause and use immediate null-return instead:

```suggestion
        if (customStatusSuggestions.length <= 0) {
            return null;
        }

        return (
            <>
                <div className='statusSuggestion__title'>
                    {formatMessage({id: 'custom_status.suggestions.title', defaultMessage: 'SUGGESTIONS'})}
                </div>
                {customStatusSuggestions}
            </>
        );
```